model/release.go

```diff
@@ -13,9 +13,12 @@
 //	limitations under the License.
 package model
 
+import "time"
+
 type Release struct {
-	Name      string
-	Artifacts []Image
+	Name      string     `json:"Name" bson:"name"`
```

Comment:
I was thinking about that too; there are two sides of it - the good one is that we don't need additional index, the bad one is that, when changing data model, it can be error prone (we already seen this when we were changing multi tenancy model). 
Also I'm not sure how regexp query on _id will behave.

Reply:
I won't insist on this change, although maintaining an additional index with the old multi-tenancy model adds a significant memory overhead for large number of tenants.